agents/LinearQ_TDistR.py

```diff
@@ -12,8 +12,8 @@ def __init__(self, state_shape, num_acts, params): # are these the initial param
     def update(self, s, sp, r, a, num_samples, done):
```

Comment:
I'm not sure this will work. We don't pass num_samples into the update function.

It would be better if num_samples was an algorithm parameter. You'd specify it in the `.json` file, then access it here via `self.params['num_samples']`.